packages/hostScriptsServices/hostScripts/recreate_dappnode.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+DNCORE_DIR="/usr/src/dappnode/DNCORE"
+DAPPMANAGER_DNCORE_FILE="$DNCORE_DIR/docker-compose-dappmanager.yml"
+LOG_DIR="/usr/src/dappnode/logs"
+RECREATE_DAPPNODE_LOG_FILE="$DNCORE_DIR/recreate_dappnode.log"
+
+# Name of the dappmanager Docker container
+DAPPMANAGER_CONTAINER_NAME="DAppNodeCore-dappmanager.dnp.dappnode.eth"
+
+# Create logs dir if not exist
+mkdir -p "$LOG_DIR"
+
+# dappmanager docker compose file does not exist, then install DAppNode
+if [ ! -f "$DAPPMANAGER_DNCORE_FILE" ]; then
+    echo "File $DAPPMANAGER_DNCORE_FILE does not exist. Installing DAppNode." | tee "$RECREATE_DAPPNODE_LOG_FILE"
+    wget -O - https://installer.dappnode.io | sudo UPDATE=true bash 2>&1 | tee "$RECREATE_DAPPNODE_LOG_FILE" 
+    return
+fi 
+
+# dappmanager docker container does not exist, then run docker compose up to dappmanager file
+if [ ! "$(docker ps -q -f name="$DAPPMANAGER_CONTAINER_NAME")" ]; then
```

Comment:
Don't you need to add flag `-a` here? If the dappmanager is stopped we might just need to restart it, not to do `compose up`

Reply:
nice catch!